cordformation/src/main/kotlin/net/corda/plugins/Baseform.kt

```diff
@@ -119,6 +119,7 @@ open class Baseform : DefaultTask() {
                 "'directory' cannot be used when 'definitionClass' is specified. Use CordformDefinition.nodesDirectory instead."
             }
             directory = cd.nodesDirectory
+            project.delete(directory)
```

Comment:
Please log this action. It will be very useful for debugging to know we're erasing a directory.